### ISSUE TITLE
Is there a way to turn all rules level to warn/error?

### ISSUE THREAD

**Issue by amankkg** [sindresorhus/eslint-plugin-unicorn#289]
E.g. as [`react-hooks` plugin](https://github.com/facebook/react/blob/dab2fdbbbd6eb560523fe2146c153f6a24e2f8dc/packages/eslint-plugin-react-hooks/src/index.js#L14) allows to do
```YAML
# rest of ESLint config
rules:
  prettier/prettier: 'error'
  react-hooks/rules-of-hooks: 'error'
  unicorn/rules-of-unicorn?: 'error'
# rest of ESLint config
```

**Comment by golopot** [comment]
This plugin could add a config that includes all rules so that people can do `extends: ["plugin:unicorn/all"]`.

**Comment by zeorin** [comment]
You can configure ESlint to treat warning as errors wrt e.g. it's exit code, or configure webpack not to stop the build on errors.

Of course, that's all-or-nothing. If you need anmore fine-grained approach you could use the `groupRules` function from [my PR on `eslint-rule-composer`](https://github.com/not-an-aardvark/eslint-rule-composer/pull/6), and make your own "new" rulenthat you can easily set an error level for.

**Comment by zeorin** [comment]
If you're not creating a shareable config, that may be overkill, actually. You could use a .js ESLint config file and set all the rules you want to group to the same variable, and change that to your desired error level. But it won't allow for inline directive comments to change them. That's what my PR is for.

**Comment by sindresorhus** [comment]
>   prettier/prettier: 'error'
  react-hooks/rules-of-hooks: 'error'

This works because those plugins only have one rule.

I think you should rather open an issue on ESLint requesting better ability to control the level for multiple rules.

**Comment by kshetline** [comment]
I'd like to see a solution to this problem as well, especially when you're extended a large pre-defined set of rules (like standard) and you don't want to essentially specify those rules all over again just to change errors to warnings. Without this capability you have to start from scratch merely to bring all errors down to warnings.

**Comment by futpib** [comment]
You might like https://github.com/IanVS/eslint-nibble, but this is not an issue for this repo or any other ESLint plugin.
